examples/tac_deploy/tac_run.sh

```diff
@@ -168,7 +168,7 @@ set_PEER(){
 	  # do nothing
 	    ;;
 	  local)
-	  	PEER=`aea get-multiaddress fetchai -c -i fetchai/p2p_libp2p:0.24.0 -u public_uri`
+	  	PEER=`aea get-multiaddress fetchai -c -i fetchai/p2p_libp2p:0.26.0 -u public_uri`
```

Comment:
Did you add this file to the automated package version update script?

Reply:
check script: it looks in this directory and *.sh files are processed.